File: Backend/SportMatchAPI/SportMatchAPI/DTO/RespondRequestDTO.cs
namespace SportMatchAPI.DTO
{
    public class RespondRequestDTO
    {
        public int InteractionId { get; set; } // ID của cái đơn xin tham gia
        public bool IsAccepted { get; set; }  // True = Đồng ý, False = Từ chối
    }
}

File: Frontend/SportMatch2/app/src/main/java/com/example/sportmatch/data/dto/RespondRequestDto.kt
package com.example.sportmatch.data.dto

import com.google.gson.annotations.SerializedName

data class RespondRequestDto(
    @SerializedName("interactionId")
    val interactionId: Int,
    @SerializedName("isAccepted")
    val isAccepted: Boolean
)